databricks_cli/scim/cli.py

```diff
@@ -0,0 +1,328 @@
+@click.option("--force", required=False, default=False)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def groups_delete_cli(api_client, group_id, group_name):
+    content = ScimApi(api_client).delete_group(group_id=group_id, group_name=group_name)
+    click.echo(pretty_format(content))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Add a user to a group')
+@click.option('--group-id', metavar='<int>', cls=OneOfOption, default=None, one_of=GROUP_OPTIONS)
+@click.option('--group-name', metavar='<string>', cls=OneOfOption, default=None,
+              one_of=GROUP_OPTIONS)
+@click.option('--user-id', metavar='<int>', cls=OneOfOption, default=None, one_of=USER_OPTIONS)
+@click.option('--user-name', metavar='<string>', cls=OneOfOption, default=None, one_of=USER_OPTIONS)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def groups_add_user_cli(api_client, group_id, group_name, user_id, user_name):
+    content = ScimApi(api_client).add_user_to_group(group_id=group_id, group_name=group_name,
+                                                    user_id=user_id,
+                                                    user_name=user_name)
+    click.echo(pretty_format(content))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Remove a user from a group.')
+@click.option('--group-id', metavar='<int>', cls=OneOfOption, default=None, one_of=GROUP_OPTIONS)
+@click.option('--group-name', metavar='<string>', cls=OneOfOption, default=None,
+              one_of=GROUP_OPTIONS)
+@click.option('--user-id', metavar='<int>', cls=OneOfOption, default=None, one_of=USER_OPTIONS)
+@click.option('--user-name', metavar='<string>', cls=OneOfOption, default=None, one_of=USER_OPTIONS)
+@click.option("--force", required=False, default=False)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def groups_remove_user_cli(api_client, group_id, group_name, user_id, user_name, force):
+    if not force and not click.confirm(
+            'Do you want to remove "{}" from group "{}"'.format(user_name, group_name)):
+        click.echo('Not removing "{}" from group "{}"'.format(user_name, group_name))
+        return
+
+    click.echo('REMOVING "{}" from group "{}"'.format(user_name, group_name))
+    content = ScimApi(api_client).remove_user_from_group(group_id=group_id, group_name=group_name,
+                                                         user_id=user_id,
+                                                         user_name=user_name)
+    click.echo(pretty_format(content))
+
+
+@click.group(context_settings=CONTEXT_SETTINGS,
+             short_help='Utility to interact with Databricks scim api.\n')
+@click.option('--version', '-v', is_flag=True, callback=print_version_callback,
+              expose_value=False, is_eager=True, help=version)
+@debug_option
+@profile_option
+@eat_exceptions
+def scim_group():
```

Comment:
suggestion: remove SCIM prefix from command groups.

Reply:
the prefix + _group is the convention across all of the commands:

```
databricks_cli/cluster_policies/cli.py:def cluster_policies_group():  # pragma: no cover
databricks_cli/clusters/cli.py:def clusters_group():  # pragma: no cover
databricks_cli/dbfs/cli.py:def dbfs_group():  # pragma: no cover
databricks_cli/groups/cli.py:def groups_group():  # pragma: no cover
databricks_cli/instance_pools/cli.py:def instance_pools_group():  # pragma: no cover
databricks_cli/jobs/cli.py:def jobs_group():  # pragma: no cover
databricks_cli/libraries/cli.py:def libraries_group():  # pragma: no cover
databricks_cli/pipelines/cli.py:def pipelines_group():  # pragma: no cover
databricks_cli/runs/cli.py:def runs_group():  # pragma: no cover
databricks_cli/secrets/cli.py:def secrets_group():  # pragma: no cover
databricks_cli/stack/cli.py:def stack_group():  # pragma: no cover
databricks_cli/tokens/cli.py:def tokens_group():  # pragma: no cover
databricks_cli/workspace/cli.py:def workspace_group():  # pragma: no cover
```